doc/source/security/encrypt-traffic.rst

```diff
@@ -12,19 +12,86 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
```

Comment:
There are 4 kind of traffic
- client-server
- sst
- ist
- replication traffic

automatic and manaul is to faciliate last 3 category.
automatic configuration (pxc-encrypt-cluster-traffic) doesn't automate client-server traffic.